_posts/2021-02-16-cockpit-238.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 238
+author: kkoukiou
+date: 2021-02-17
+tags: cockpit linux
+slug: cockpit-238
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 238.
+
+### Updates: Show the list of services and applications that are outdated
+
+Previously, we required a system reboot after every update. We now use [Tracer](http://tracer-package.com/)
+to display the list of services and applications that are outdated and need to be restarted or require a system reboot.
+
+If there are not such services not system reboot will be suggested.
+
+![Update no restart](/images/update-no-restart.png)
+
+If there are such services, the user will be asked to either to restart these or schedule a system reboot.
+
+![Update with restart](/images/update-with-restart.png)
+
+### Web server: Do not change permissions of administrator provided certificates
```

Comment:
```suggestion
### Web server: Preserve permissions of administrator-provided certificates
```

Reply:
Preserve here, implies that we are actually doing something no?

On the other hand this was really a bug where i changed the certificated permission when we should not do that, therefore i prefer the previous option that martin suggested, since it shows that we don't change the original permissions.

Not strong opinion here though.

Reply:
Preserve says we're not changing it.

Hyphen is needed in this usage.